### DISCUSSION TITLE
Using deployer from windows 10

### DISCUSSION THREAD

**Discussion by etshy** [deployphp/deployer#2698] [Help needed]
I just discovered this package and tried to use it on my local machine, windows 10.
I installed ssh windows service, to be able to use pub key, gen a key, etc.

But it's still impossible to use it.
I have this error when deployer try to connect to my remote host
```
The command "ssh -N -A -o ControlMaster=auto -o ControlPersist=60 -o "ControlPath=C:\Users\Etshy/.ssh/deployer_user@X.X.X.X" user@X.X.X.X" failed.  
                                                                                                                                                                    
  Exit Code: 255(Unknown error)
```

First thing, it's unclear you have to create a file/config in `C:\Users\Etshy/.ssh/deployer_user@X.X.X.X` (I don't really know what this is, I never saw that kind of ssh command and options.

Is there an y way to make it work for a Windows ? or do I absolutely have to use it on linux ?

edit : After a few test, from the ssh command, it seems the `-N` option is too much, by removing it the connection works (I had to add -p port too, because  my remote doesn't use default ssh port)

**Comment by antonmedv** [comment]
Try disabling ssh_multiplexing.

**Reply by isaacdarcilla** [reply]
This works.

**Comment by acharseth** [comment]
Even if you can get SSH work I think you will get a challenge with the creation of symbolic links, which is not supported in Windows out of the box (security issue).
